### PR TITLE
Added AuthorEntity

### PR DESCRIPTION
**Pull request by praveen4892**
- Added required fields.

### CODE REVIEW

File: src/main/java/com/dev/sunny/model/Author.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.dev.sunny.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+@Getter
```
Comment (SunnysGitAccount) [inline src/main/java/com/dev/sunny/model/Author.java:15]: Add indentation

File: src/main/java/com/dev/sunny/model/Author.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@Setter
+@ToString
+public class Author {
```
Comment (SunnysGitAccount) [inline src/main/java/com/dev/sunny/model/Author.java:20]: Add entity annotation

File: src/main/java/com/dev/sunny/model/Author.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
```
Comment (SunnysGitAccount) [inline src/main/java/com/dev/sunny/model/Author.java:21]: Add @Id, @GeneratedValue annotations

**Review by SunnysGitAccount — CHANGES_REQUESTED** [review]
PR reviewed. There are some changes requested as the code change is incorrect and Author class is currently not an Entity.

File: src/main/java/com/dev/sunny/model/Author.java (hunk 2 above)
Comment (praveen4892) [reply src/main/java/com/dev/sunny/model/Author.java:20]: Added annotation

File: src/main/java/com/dev/sunny/model/Author.java (hunk 3 above)
Comment (praveen4892) [reply src/main/java/com/dev/sunny/model/Author.java:21]: Added both

File: src/main/java/com/dev/sunny/model/Author.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+@Entity
+public class Author {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Comment (SunnysGitAccount) [inline src/main/java/com/dev/sunny/model/Author.java:23]: Add access specifiers to every attribute.

File: src/main/java/com/dev/sunny/model/Author.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    Long id;
+    String name;
+    String email;
+    List<String> publications;
+
```
Comment (SunnysGitAccount) [inline src/main/java/com/dev/sunny/model/Author.java:28]: Add constructor with all attribute

 - Should be without id attribute.
 - Use @NoArgsConstructor to provide default constructor.

Comment (SunnysGitAccount) [inline src/main/java/com/dev/sunny/model/Author.java:28]: Add equals and hash code implementation
 
- Use IntelliJ Default implementation (Recommendation).
- Use only id as not null member.

**Review by SunnysGitAccount — CHANGES_REQUESTED** [review]
Please update the class with requested changes.

File: src/main/java/com/dev/sunny/model/Author.java (hunk 4 above)
Comment (praveen4892) [reply src/main/java/com/dev/sunny/model/Author.java:23]: Done

File: src/main/java/com/dev/sunny/model/Author.java (hunk 5 above)
Comment (praveen4892) [reply src/main/java/com/dev/sunny/model/Author.java:28]: Resolved above changes

Comment (praveen4892) [reply src/main/java/com/dev/sunny/model/Author.java:28]: Done

**Review by SunnysGitAccount — APPROVED** [review]
Good job completing PR Review changes. All changes are approved.

**Pull request merged**